nekoyume/Assets/_Scripts/UI/Module/Item/EquipmentSlots.cs

```diff
@@ -168,11 +169,9 @@ public bool TryGetToEquip(Equipment equipment, out EquipmentSlot slot)
 
             if (itemSubType == ItemSubType.Ring)
             {
-                var itemId = equipment.ItemId;
-                slot = typeSlots.FirstOrDefault(e =>
-                           !e.IsEmpty &&
-                           e.Item is ItemUsable itemUsable &&
-                           itemUsable.ItemId.Equals(itemId))
+                slot = typeSlots.OrderBy(e => CPHelper.GetCP((ItemUsable) e.Item,
+                               Game.Game.instance.TableSheets.CostumeStatSheet))
```

Comment:
Is it okay to remove previous filtering condition?
Also, use `public static int GetCP(ItemUsable itemUsable)`.

Reply:
I'll check and fix it.